Games/2048/main.cpp

```diff
@@ -75,6 +75,9 @@ int main(int argc, char** argv)
     app_menu.add_action(GUI::Action::create("New game", { Mod_None, Key_F2 }, [&](auto&) {
         game.reset();
     }));
+    app_menu.add_action(GUI::Action::create("Undo", { Mod_None, Key_U }, [&](auto&) {
+        game.undo();
+    }));
```

Comment:
Let's use `GUI::CommonActions::make_undo_action` so we get the right icon and shortcut :)

Reply:
Oh we have that? neat!